lighthouse-cli/bin.ts

```diff
@@ -143,6 +147,10 @@ if (!cliFlags.outputPath && cliFlags['output-path']) {
   cliFlags.outputPath = cliFlags['output-path'];
 }
 
+if (!cliFlags.maxWaitForLoad && cliFlags['max-wait-for-load']) {
```

Comment:
done, from the looks of it google3 is still on lighthouse ~1.1 so we likely will have quite a few other issues to iron out when they want to upgrade anyway...